src/components/Garden/Feed/ProposalDescription.js

```diff
@@ -14,6 +14,7 @@ function ProposalDescription({ proposal, onSelectProposal }) {
         margin-bottom: ${3 * GU}px;
         ${textStyle('body1')};
         text-decoration: underline;
+        overflow-wrap: anywhere;
```

Comment:
What do you think of also giving the styles to limit the description lines to 3?
```suggestion
        overflow-wrap: anywhere;
        display: -webkit-box;
        -webkit-line-clamp: 3;
        -webkit-box-orient: vertical;
        overflow: hidden;
```

Reply:
> What do you think of also giving the styles to limit the description lines to 3?

the code works correctly, (obviously), i don't have a strong opinion about it, could some proposals need a longer title?

Reply:
Yeah i mean, if proposals do have a very long title, it would be nice to limit it to just 3 lines, so it doesn{t take a lot of space in the feed. Does that make sense?

Reply:
Yeah, looks great!